charts/prometheus-postgres-exporter/README.md

```diff
@@ -40,11 +40,48 @@ _See [helm uninstall](https://helm.sh/docs/helm/helm_uninstall/) for command doc
 
 ## Upgrading
 
+### To 3.0.0
+
+This release introduces changes to accommodate Postgres 13 or newer versions by default.
+Older Postgres instances have now to overwrite the `pg_stat_statements` query of `config.queries` with following
+selection:
+
+```postgresql
```

Comment:
can we put this in a single line so copieng is easier?

```postgresql
SELECT t2.rolname, t3.datname, queryid, calls, total_time / 1000 as total_time_seconds, min_time / 1000 as min_time_seconds, max_time / 1000 as max_time_seconds, mean_time / 1000 as mean_time_seconds, stddev_time / 1000 as stddev_time_seconds, rows, shared_blks_hit, shared_blks_read, shared_blks_dirtied, shared_blks_written, local_blks_hit, local_blks_read, local_blks_dirtied, local_blks_written, temp_blks_read, temp_blks_written, blk_read_time / 1000 as blk_read_time_seconds, blk_write_time / 1000 as blk_write_time_seconds FROM pg_stat_statements t1 JOIN pg_roles t2 ON (t1.userid=t2.oid) JOIN pg_database t3 ON (t1.dbid=t3.oid) WHERE t2.rolname != 'rdsadmin' AND queryid IS NOT NULL
```

Reply:
The linter prevents this. https://github.com/prometheus-community/helm-charts/pull/2021/commits/3ebf12bee652696559b709b20df5d493946da320